### ISSUE TITLE
Standard for per investigator identifier of numerical values (such as health/clues/etc)

### ISSUE THREAD

**Issue by PlaidOsyluth** [Kamalisk/arkhamdb-json-data#325]
Is there a standard way to be able to indicate that a particular field (i.e. clues) should be 'per investigator' in the json?

While I don't see a way in the card_schema.json to do this, I do see that in the tfa_encounter.json, a 'health_per_investigator' boolean field was used for the Harbinger of Valusia. Is this a standard field we should expect across all numerical fields?  I ask because I don't see a similar "clues_per_investigator" field for locations that have variable clues in that same tfa_encounter.json.

Would it make more sense to have a single field that designates all fields which should be adjusted by number of investigators?  i.e. "per_investigator_fields":"clues,health" 
(I understand that it will be highly unlikely to have a template of a location with health, or an enemy with clues, but it would also allow for flexibility if fields we aren't expecting to have 'per investigator' suddenly gained them.  (i.e. enemy_fight - *shutter*))

**Comment by zzorba** [comment]
For clues it’s axtually inverted, since most are per investigator some have clues_fixed to indicate they aren’t per investigator

**Comment by PlaidOsyluth** [comment]
Thanks for the quick response. Are there any such fields for any of the other numerical values (i.e. doom/enemy_fight)? Should these be added to the card_schema?

**Comment by zzorba** [comment]
So today I think its just clues and enemy_health, I don't believe the game has yet done a per_investigator fight/evade, right?.  For enemies that have an 'X' value in their fight value, the convention so far seems to be to set it to a 'negative' value.

I don't think the per-investigator doom is currently captured in the system, I just took a look at the echoes of the past agenda and it doesn't seem like its been called out.

It does seem like some kind of API change would be needed to capture that.

**Comment by drawntotheflame** [comment]
Yeah, when Echoes of the Past came out, @Kamalisk suggested he'd take a look when he had time but it wasn't a priority - which I'd agree, it isn't, currently, in the grand scheme of things!

**Comment by tjanu** [comment]
To rekindle this conversation - there was a card spoiled for the Machinations through Time scenario that will have a new field be scaled per investigator - maybe this is the opportunity to think about solutions for this issue?
<details>
  <summary>Spoiler - the newly scaled field</summary>
  
  There is a location with per investigator shroud
  
</details>
